app/main.py

```diff
@@ -2,0 +1,16 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Invalid command format. "
+                         "Expected: 'cp file.txt new_file.txt'")
+
+    old_file_name = parts[1]
+    new_file_name = parts[2]
```

Comment:
Consider using more descriptive variable names for `old_file_name` and `new_file_name` to improve code readability. For example, `source_file_name` and `destination_file_name`.